Gruntfile.js

```diff
@@ -179,7 +179,11 @@ module.exports = function (grunt) {
     wiredep: {
       app: {
         src: ['<%= yeoman.app %>/index.html'],
-        exclude: ['bower_components/bootstrap-sass-official/assets/javascripts/bootstrap.js'],
+        exclude: [
+            'bower_components/bootstrap-sass-official/assets/javascripts/bootstrap.js',
+            // Leaflet provided by cartodb.js
+            'bower_components/leaflet-dist/*'
```

Comment:
Since `leaflet-draw` is already included, I guess we probably don't need any more plugins, though.